pkg/stanza/fileconsumer/config.go

```diff
@@ -83,7 +83,7 @@ func (c Config) Build(logger *zap.SugaredLogger, emit EmitFunc) (*Manager, error
 	}
 
 	// Ensure that splitter is buildable
-	factory := newMultilineSplitterFactory(c.Splitter.EncodingConfig, c.Splitter.Flusher, c.Splitter.Multiline)
```

Comment:
This looks like a good change. Would you please open a separate PR for it though?

Reply:
Filed https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/18303